src/json/util.h

```diff
@@ -51,7 +51,7 @@ typedef enum {
     JSONUTIL_ADDITION_OVERFLOW,
     JSONUTIL_EMPTY_JSON_OBJECT,
     JSONUTIL_EMPTY_JSON_ARRAY,
-    JSONUTIL_INDEX_OUT_OF_ARRAY_BOUNDARIES,
+    JSONUTIL_INDEX_OUT_OF_ARRAY_BOUNDS,
```

Comment:
I'm hoping this is private enough... otherwise, I'm happy to drop this or any other change...